llms/ollama/internal/ollamaclient/types.go

```diff
@@ -49,7 +49,7 @@ type Message struct {
 type ChatRequest struct {
 	Model     string     `json:"model"`
 	Messages  []*Message `json:"messages"`
-	Stream    *bool      `json:"stream,omitempty"`
+	Stream    bool       `json:"stream,omitempty"`
```

Comment:
This makes the `omitempty` meaningless, because bools can't be "empty".

Reply:
that's not quite right, in Go json encoding: `The "omitempty" option specifies that the field should be omitted from the encoding if the field has an empty value, defined as false, 0, a nil pointer, a nil interface value, and any empty array, slice, map, or string.`

Reply:
hmm, then perhaps I'm mistaken... I recall seeing `"field": false` in JSON output despite `omitempty`, but I haven't tested it so I may be misremembering :)